Services/ContentExport.php

```diff
@@ -222,6 +222,10 @@ public function generateTable(ContentType $contentType, $format)
     private function addRow($values, $handle, $format)
     {
         if ($format == 'csv') {
+            $values = array_map(function ($value) {
+                return str_replace(["\n", "\r", "\r\n", "\n\r"], '', $value);
```

Comment:
You are removing newlines which could be important info for the user. Also the first two entries also replace everyting in the last two entries so those are not needed.